modules/installing-ocp-agent-disk-partition.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
```

Comment:
Given that the content should be identical to the UPI and IPI methods, is there a way they can all share a single module so that the docs can't become out of sync?

Ideally we'd have a separate section of the manual for this that all 3 methods referred to, but we're not going to solve that in this PR :)

Reply:
The original file with this content was quite large and included other content that didn't seem appropriate in the agent docs, as far as I can tell, so I initially wanted to create a new file for simplicity's sake. 

However, if you'd like to use the same module found in IPI and UPI, I'll delete this file and use conditional statements in the original module so that only this content (minus the UPI stuff you asked me to trim out) shows up in the agent docs.